### PR TITLE
fix(angular/i18n): fix translations not used on production

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/angular/core/common-behaviors/common.module.ts [hunk 1]
```diff
@@ -32,6 +34,18 @@ export interface GranularSanityChecks {
   version: boolean;
 }
 
+/**
```
Comment (kyubisation) [inline src/angular/core/common-behaviors/common.module.ts:37]: Should we maybe move this to after the module to the after functions?

File: src/angular/core/common-behaviors/common.module.ts [hunk 2]
```diff
@@ -75,6 +89,8 @@ export class SbbCommonModule {
         }
       }
     }
+
+    patch$localize();
```
Comment (kyubisation) [inline src/angular/core/common-behaviors/common.module.ts:93]: Should this be moved to the other setup call before the if check?

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Review by mhaertwig — APPROVED** [review]
LGTM

**Pull request merged**